<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="setting_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>637</width>
     <height>145</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="autoScroll">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashDotLine</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <row>
    <property name="text">
     <string>Auto</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>A</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>B</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>C</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>IP</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>BUFFER</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>PORT</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Master ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Slave ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Serial Number</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>192.168.0.201</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>3060</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>502</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>1</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="4">
    <property name="text">
     <string>1</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="5">
    <property name="text">
     <string>COM5</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="1" column="0">
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="1" column="1">
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="1" column="2">
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="1" column="3">
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="1" column="4">
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="1" column="5">
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="2" column="0">
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="2" column="1">
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="2" column="2">
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="2" column="3">
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="2" column="4">
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="2" column="5">
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string/>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="3" column="1">
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="3" column="2">
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="3" column="3">
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="3" column="4">
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="3" column="5">
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
  </widget>
  <widget class="QRadioButton" name="auto_setting_btn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Setting</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="A_setting_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>A Setting</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="B_setting_btn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>B Setting</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="C_setting_btn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>C Setting</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>170</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>세팅 저장</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>테스트</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
